frame/membership/src/lib.rs

```diff
@@ -79,7 +81,7 @@ pub mod pallet {
 		///
 		/// This is used for benchmarking. Re-run the benchmarks if this changes.
 		///
-		/// This is not enforced in the code; the membership size can exceed this limit.
+		/// This is enforced in the code; the membership size can not exceed this limit.
 		type MaxMembers: Get<u32>;
```

Comment:
Not sure if we intentionally made it so that it's unenforceable, someone else with more context needs to review this to ensure we're not breaking anything here, and if so, we'd have to use a `WeakBoundedVec` rather than a `BoundedVec`.

Reply:
no, this is just because it was not bounded. We want to enforce this limit.